ads/adman.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adman(global, data) {
+  const script = document.createElement('script');
```

Comment:
To confirm, are any of these `data` parameters required? If so, you may want to check them using `validateDataExists`.

Also, ideally you'd check the allowed set of data parameters using `checkData`. This would help your customers to avoid issues with misspellings, etc.